### DISCUSSION TITLE
希望在Mac OS上也增加自动启动功能

### DISCUSSION THREAD

**Discussion by shep2008** [localsend/localsend#1100] [L-简体中文]
希望在Mac OS上也增加自动启动功能，随系统启动
![Snipaste_2024-01-21_23-04-42](https://github.com/localsend/localsend/assets/42241740/e74d2756-3975-4513-a7d6-b5b522462b9c)

**Comment by junceo** [comment]
可以在系统设置->登录项->登录时打开 添加。
